installer/installer.py

```diff
@@ -994,6 +1068,29 @@ def ask_for_cosmovisor(self):
             self.is_cosmo_needed = False
         else:
             failure_exit(f"Invalid input provided during installation.")
+    
+    def ask_for_cosmovisor_bump(self):
+        self.exec("echo 'export DAEMON_NAME=cheqd-noded' >> ~/.bashrc")
+        self.exec(f"echo 'export DAEMON_HOME={self.home_dir}/.cheqdnode' >> ~/.bashrc")
+        self.exec(f"echo 'export DAEMON_DATA_BACKUP_DIR={self.home_dir}/.cheqdnode' >> ~/.bashrc")
```

Comment:
1. The installer might be executing as ANY user, and therefore you need to ensure it's put in THAT user's environment variables, but also in the cheqd user's environment variables.
2. Use Python-native libraries to write to the file. You don't actually know whether these files are writeable or not. https://stackoverflow.com/questions/17657686/is-it-possible-to-set-an-environment-variable-from-python-permanently

Reply:
This should be done in bump_cosmovisor, NOT in ask_for_cosmovisor

Reply:
@ankurdotb Regarding checking file permission I tried this function, but it always returns false for both these 2 file paths I pass.
```is_file_writable("/home/cheqd/.bashrc")```
```is_file_writable(os.path.expanduser("~/.bashrc"))```
```python3
def is_file_writable(self, path_to_file):
        if os.path.exists(path_to_file) and os.path.isfile(path_to_file): 
            return os.access(path_to_file, os.W_OK)
        return False
```
And since the `glibc` issue is not related to that, I didn't want to so much time on it.